src/Relation/Nullary.agda

```diff
@@ -26,6 +26,11 @@ infix 2 _because_
 ¬_ : ∀ {ℓ} → Set ℓ → Set ℓ
 ¬ P = P → ⊥
 
+-- level polymorphic version of ¬
+-- i.e. ¬ 'i'ndexed by a level.
+¬ᵢ : ∀ {ℓ} (b : Level) → Set ℓ → Set (ℓ ⊔ b)
+¬ᵢ b P = P → Lift b ⊥
```

Comment:
Now that I think about it, given that we have `Relation.Binary.Indexed` and `Relation.Unary.Indexed`, this should probably live in `Relation.Nullary.Indexed`? Then we wouldn't even have to have the subscript?

Reply:
Makes sense.